### ISSUE TITLE
Deploy failing with: Command "argo:build" not found 

### ISSUE THREAD

**Issue by ChmilevFA** [argoapp-live/beta-testing#57]
## Describe the bug
While trying to deploy the app I'm getting the error "Command "argo:build" not found"

## Criticality
`Critical` as it's blocking app deployment completely

## To Reproduce

1. Go to new deployment and set up configs
![image](https://user-images.githubusercontent.com/17650184/169316251-1b491ca5-a44f-4619-bc41-b34a128ff2ea.png)
with ROOT DIRECTORY: frontend
repository: https://github.com/hackmoney-uf/defi-statements (in case it is fixed already use https://github.com/hackmoney-uf/defi-statements/commit/5fcd95ae119dbdd8c3ce746506f8df3debf8ef62 as last commit)
2. Try to deploy

## Expected behavior
App deployed

## Screenshots
Config:
![image](https://user-images.githubusercontent.com/17650184/169316251-1b491ca5-a44f-4619-bc41-b34a128ff2ea.png)
Logs:
![image](https://user-images.githubusercontent.com/17650184/169316427-28bda71a-4e26-4aac-ac00-274879e359b0.png)

## Desktop (please complete the following information):

 - OS: Any
 - Browser: Any
 - Version: Any
 - Location: Any

## Additional context
Seems like `rekpero#3898` already on it, but to fix it I had to add
```
module.exports = {
  images: {
    loader: 'akamai',
    path: '',
  },
}
```
to my next.js app

## Your Details

Discord: Fedor#0110

**Comment by rekpero** [comment]
Though It has been fixed, thanks for reporting it @ChmilevFA

**Comment by ChmilevFA** [comment]
@rekpero probably worth to add somewhere in docs that user have to modify next.js config

**Comment by rekpero** [comment]
Yes @ChmilevFA, we are preparing a particular section for the next js deployment via spheron. We will add these extra steps needed for an app to deploy successfully via Spheron.
